src/main/java/org/springframework/samples/petclinic/rest/importcsv/ImportCSV.java

```diff
@@ -23,6 +23,109 @@
 @RequestMapping("api/import")
 public class ImportCSV {
 
+    class ParsedEntity {
+
+        private String field;
+        private int pos;
+
+        /***
+         * contructor
```

Comment:
I guess this comments were not finished because of time reasons...just want to comment that a more meaning full text would help why you added this class